<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Smoke Test Suite" duration-ms="25874" started-at="2021-11-19T21:54:09Z" finished-at="2021-11-19T21:54:35Z">
    <groups>
    </groups>
    <test name="LoginPage Tests Chrome" duration-ms="25874" started-at="2021-11-19T21:54:09Z" finished-at="2021-11-19T21:54:35Z">
      <class name="sfdc.tests.Login">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="setUp" is-config="true" duration-ms="2150" started-at="2021-11-19T16:54:07Z" finished-at="2021-11-19T16:54:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="initializePageObjects()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="initializePageObjects" is-config="true" duration-ms="13" started-at="2021-11-19T16:54:09Z" finished-at="2021-11-19T16:54:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializePageObjects -->
        <test-method status="PASS" signature="launchApp()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="launchApp" is-config="true" duration-ms="2691" started-at="2021-11-19T16:54:09Z" finished-at="2021-11-19T16:54:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="FAIL" signature="loginTC01()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="loginTC01" duration-ms="620" started-at="2021-11-19T16:54:12Z" finished-at="2021-11-19T16:54:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	expected [Please enter your password.] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Please enter your password.] but found []
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at sfdc.tests.Login.loginTC01(Login.java:145)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTC01 -->
        <test-method status="PASS" signature="launchApp()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="launchApp" is-config="true" duration-ms="361" started-at="2021-11-19T16:54:12Z" finished-at="2021-11-19T16:54:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="PASS" signature="loginTC02()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="loginTC02" duration-ms="5747" started-at="2021-11-19T16:54:13Z" finished-at="2021-11-19T16:54:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTC02 -->
        <test-method status="PASS" signature="launchApp()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="launchApp" is-config="true" duration-ms="194" started-at="2021-11-19T16:54:19Z" finished-at="2021-11-19T16:54:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="PASS" signature="loginTC04b()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="loginTC04b" duration-ms="10058" started-at="2021-11-19T16:54:19Z" finished-at="2021-11-19T16:54:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTC04b -->
        <test-method status="PASS" signature="launchApp()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="launchApp" is-config="true" duration-ms="175" started-at="2021-11-19T16:54:29Z" finished-at="2021-11-19T16:54:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="PASS" signature="loginTC04a()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="loginTC04a" duration-ms="1150" started-at="2021-11-19T16:54:29Z" finished-at="2021-11-19T16:54:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTC04a -->
        <test-method status="PASS" signature="launchApp()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="launchApp" is-config="true" duration-ms="185" started-at="2021-11-19T16:54:30Z" finished-at="2021-11-19T16:54:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method status="FAIL" signature="loginTC03()[pri:1, instance:sfdc.tests.Login@1e397ed7]" name="loginTC03" duration-ms="3976" started-at="2021-11-19T16:54:30Z" finished-at="2021-11-19T16:54:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [] but found [sana@teckarch.com]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [] but found [sana@teckarch.com]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at sfdc.tests.Login.loginTC03(Login.java:207)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTC03 -->
        <test-method status="PASS" signature="finishTest()[pri:0, instance:sfdc.tests.Login@1e397ed7]" name="finishTest" is-config="true" duration-ms="277" started-at="2021-11-19T16:54:35Z" finished-at="2021-11-19T16:54:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- finishTest -->
      </class> <!-- sfdc.tests.Login -->
    </test> <!-- LoginPage Tests Chrome -->
  </suite> <!-- Smoke Test Suite -->
</testng-results>
